### ISSUE TITLE
[1.20.1] Framed Drawers Don't Support 128-bit textures

### ISSUE THREAD

**Issue by thepriz** [Buuz135/FunctionalStorage#270]
**DO NOT REPORT AN ISSUE IF YOU HAVEN'T TESTED THE ISSUE WITH THE LATEST VERSION IN CURSEFORGE**

**Functional Storage Version:** 
1.20.1-1.2.10

**Titanium Version:**
1.20.1-3.8.27

I used to be able to use my 128-bit textures with framed drawers and it would render perfectly and fit in with all my textures. now it somehow is only using part of the texture to render the drawer.

I would expect when I use a 128-bit texture pack and use that texture on a framed drawer it would render with the 128-bit texture.

**Comment by Buuz135** [comment]
Can you attach a screenshot?

**Comment by thepriz** [comment]
> Can you attach a screenshot?

Yes, will try to get what it looked like before it didn't work and what it looks like now.

**Comment by thepriz** [comment]
![2024-04-10 128-bit texture working](https://github.com/Buuz135/FunctionalStorage/assets/34556672/8ce27114-1c80-497f-98cc-b6e1828a9f56)
![2024-04-10 164-bit texture not working](https://github.com/Buuz135/FunctionalStorage/assets/34556672/8dba74c0-f2bf-4a18-b6d3-f4f0883a66b6)

**Comment by Buuz135** [comment]
Does it happen only when you change resource packs?

**Comment by thepriz** [comment]
It is the same resource pack just one is in 1.19.2 and the second one is in 1.20.1
